extensions/sql-database-projects/src/projectProvider/projectProvider.ts

```diff
@@ -211,11 +212,15 @@ export class SqlDatabaseProjectProvider implements dataworkspace.IProjectProvide
 	/**
 	 * Gets the Database Schema Provider version for a SQL project
 	 */
-	async getProjectDatabaseSchemaProvider(projectFilePath: string): Promise<string> {
+	public async getProjectDatabaseSchemaProvider(projectFilePath: string): Promise<string> {
 		return await this.projectController.getProjectDatabaseSchemaProvider(projectFilePath);
 	}
 
-	async generateProjectFromOpenApiSpec(options?: sqldbproj.GenerateProjectFromOpenApiSpecOptions): Promise<sqldbproj.ISqlProject | undefined> {
+	public async generateProjectFromOpenApiSpec(options?: sqldbproj.GenerateProjectFromOpenApiSpecOptions): Promise<sqldbproj.ISqlProject | undefined> {
 		return await this.projectController.generateProjectFromOpenApiSpec(options);
 	}
+
+	getPublishToDockerSettings(project: sqldbproj.ISqlProject): Promise<sqldbproj.IPublishToDockerSettings | undefined> {
```

Comment:
explicit scope please

Reply:
Getting ahead of me =) I fixed a couple here, then noticed that there were a bunch in this file (and some other minor issues) so decided to do them in a separate PR. Should have reverted the ones here but not going to bother with that at this point I think...